vxvas2nets/vas2nets_sms.py

```diff
@@ -179,19 +219,42 @@ def handle_outbound_message(self, message):
             returnValue((yield self.handle_outbound_fail(message, resp)))
 
     @inlineCallbacks
+    def handle_send_timeout(self, message):
+        yield self.add_status(
```

Comment:
Nack-ing would cause us to retry, right? Avoided it for that reason, since retrying http requests sounds like a rabbit hole. For e.g. messages continuously being retried and no backing-off of retries. Am I misunderstanding things?